cypress/e2e/seededFlows/articleFlows/commentModerationTools.spec.js

```diff
@@ -30,83 +23,81 @@ describe('Moderation Tools for Comments', () => {
 
       cy.get('@trustedUser').then((trustedUser) => {
         cy.loginAndVisit(trustedUser, '/series_user/series-test-article-slug');
+        cy.intercept('/notifications/counts').as('notificationCount');
+        cy.intercept('/async_info/base_data').as('baseData');
+        cy.intercept('/ahoy/visits').as('ahoy');
```

Comment:
Ah - I'm a bit surprised that these weren't getting intercepted as they should? We're _supposed_ to be [automatically waiting for these to complete](https://github.com/forem/forem/blob/09828853f6cc0e07da74f4da885dfa93f5204662/cypress/util/networkUtils.js) whenever we call [`loginAndVisit`](https://github.com/forem/forem/blob/800ff397e0172fb0b165d76526c5803ab202cc22/cypress/support/commands.js#L45-L71)